internal/controllers/machinedeployment/machinedeployment_controller.go

```diff
@@ -37,6 +37,7 @@ import (
 
```

Comment:
(just had to put the comment somewhere)

@ykakarap Please follow up on the remaining findings on https://github.com/kubernetes-sigs/cluster-api/pull/7921